src/components/ReadMore/index.js

```diff
@@ -44,7 +44,7 @@ const ReadMore = (props) => {
     <View onLayout={handleLayoutWidth} style={props.containerStyle}>
       {isFinishSetLayout ? (
         <TouchableOpacity testID="finishLayout" onPress={props.onPress}>
-          <Text style={props.textStyle}>
+          <Text style={[{color: COLORS.gray400}, props.textStyle]}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The hardcoded color `COLORS.gray400` might not be consistent with the rest of the application's theme. It would be better to use a color from a centralized theme file or pass it as a prop, so that it can be changed dynamically based on the theme.
```diff
- <Text style={[{color: COLORS.gray400}, props.textStyle]}>
+ <Text style={[{color: props.textColor || COLORS.gray400}, props.textStyle]}>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->